### PR TITLE
Improving error message in case the Cloud Billing API is not enabled

### PR DESCRIPTION
**Pull request by mwatney-mars**
Changed the error message from:

    print("Permission Denied, check project level permission.")
to:

    print("Permission Denied, check project level permission or if the Cloud Billing API is enabled")
as the pervious one is not indicating the issue may be related to the Cloud Billing API to be disable.

### CODE REVIEW

File: examples/billboard/billboard.py [hunk 1]
```diff
@@ -297,7 +297,7 @@ def main(argv):
         project_billing_info = billing.CloudBillingClient(
         ).get_project_billing_info(name=project_id_temp)
     except PermissionDenied:
-        print("Permission Denied, check project level permission.")
+        print("ermission Denied, check project level permission or if the Cloud Billing API is enabled.")
```
Comment (shakeebshams) [inline examples/billboard/billboard.py:300]:
```suggestion
        print("Permission Denied, check project level permission or if the Cloud Billing API is enabled.")
```

**Review by agold-rh — DISMISSED** [review]

**Comment by agold-rh** [comment]
@mwatney-mars Python 3 formatting check failed. Look in helpers/check_format.sh for the standard Python 3 format checking options.

**Comment by yashwantm** [comment]
yapf formatting limit the print string length can not exceed 63 char. reduce the length to pass the checks.

**Comment by agold-rh** [comment]
@mwatney-mars  Formatting checks are failing in examples/billboard/billboard.py. See the earlier comments for suggestions.

**Comment by agold-rh** [comment]
@mwatney-mars See the "Resolve conflicts" messages.

**Comment by agold-rh** [comment]
@mwatney-mars Formatting checks are still failing in billboard.py. See the Oct 9th comments for how to verify formatting.

**Pull request closed without merging**